dockstore-webservice/src/main/java/io/dockstore/webservice/resources/UserResource.java

```diff
@@ -911,8 +911,7 @@ public Set<Entry> getStarredServices(@ApiParam(hidden = true) @Parameter(hidden
      * @return a user's starredEntries for desired entryTypes
      */
     public Set<Entry> getStarredEntries(User user, Set<EntryType> desiredTypes) {
```

Comment:
`EntryTypeMetadata` is sort of a souped-up webservice-only version of the dockstore-common `EntryType`, so an alternative implementation would be to use `EntryTypeMetadata` to denote the particular entry types (instead of `EntryType`).  Then we don't need the `static` conversion function `entryType2class`.  Not sure if it's the better solution, but food for thought.

(The above has me wondering if it is somehow possible for `EntryTypeMetadata` to extend `EntryType`, but that's a discussion for later.)